static/js/src/advantage/react/components/Subscriptions/SubscriptionList/SubscriptionList.tsx

```diff
@@ -1,17 +1,31 @@
-import { Card } from "@canonical/react-components";
 import React from "react";
 
+import ListCard from "./ListCard";
+import ListGroup from "./ListGroup";
+
 const SubscriptionList = () => (
-  <div className="p-subscribe__list">
-    <div className="p-subscribe__list-scroll">
-      <Card>Card</Card>
-      <Card>Card</Card>
-      <Card>Card</Card>
-      <Card>Card</Card>
-      <Card>Card</Card>
-      <Card>Card</Card>
-      <Card>Card</Card>
-      <Card>Card</Card>
+  <div className="p-subscriptions__list">
+    <div className="p-subscriptions__list-scroll">
+      <ListGroup title="Ubuntu Advantage">
+        <ListCard
+          created="12.02.2021"
+          expires="23.04.2022"
+          features={["ESM Infra", "livepatch", "24/5 support"]}
+          machines={10}
+          period="Annual"
+          title="Lorem ipsum dolor sit amet, consectetur adipiscing elit"
+        />
+      </ListGroup>
+      <ListGroup title="free personal token">
+        <ListCard
+          created="12.02.2021"
+          expires="Never"
+          features={["ESM Infra", "livepatch"]}
+          machines={3}
+          period="Free"
```

Comment:
I know it will be a period _most_ of the time but this feels weird.

Reply:
Good point, I've renamed this to `label`. Maybe a bit too generic, but hopefully less confusing.